recipes/kraken-all/meta.yaml

```diff
@@ -0,0 +1,13 @@
+package:
+  name: kraken-all
+  version: "0.10.5beta"
+requirements:
+  build:
+    - kraken ==0.10.5beta
```

Comment:
It will, but this is about run requirements. Why fixing the version of the build requirement. Further, this is a metapackage, it should not have a build requirement at all...